src/components/layouts/Settings.tsx

```diff
@@ -39,102 +39,107 @@ interface AdvanceSetting {
 
 export function Settings(props: Props) {
   const [t] = useTranslation();
+
   const location = useLocation();
 
+  const doesPathNameIncludes = (route: string) => {
+    return location.pathname.includes(route);
+  };
```

Comment:
I don't think we need a separate function for this. See comment below.

Reply:
removed